docs/source/strategies/collaborative_training_expert.rst

```diff
@@ -0,0 +1,84 @@
+:orphan:
+
+.. _collaborative_training_expert:
+
+Training on unreliable mixed GPUs across the internet (Expert)
+==============================================================
+
+Using Compression to Optimize communications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Below are some ways to reduce communication when training collaboratively. As the model sizes get larger, bottlenecks in communication become more apparent.
+
+Compress Gradients & State
+""""""""""""""""""""""""""
+
+Hivemind allows you to compress gradients and states before sending them to other machines. This helps reduce the communication overhead substantially when training across the internet.
```

Comment:
I just added a note to mention this may affect convergence if going from Float32 -> Float16. 

I haven't done enough research around intra-node performance, but hivemind always uses the network adapter (local DDP support is a later thing to work on).